src/code/fv-components/stable-drake.js

```diff
@@ -0,0 +1,24 @@
+import React, {PropTypes} from 'react';
+import OrganismView from '../components/organism';
+
+const StableDrakeView = ({org, id, width, height, style, onClick}) => {
```

Comment:
This is a good point- since the overlay isn't reusable, I like just wrapping this in the stable instead